services/processor/processor/processor.py

```diff
@@ -244,8 +246,47 @@ def start_processing(self):
                             self,
                             payload,
                         )
-
+                    except MissingParentError:
```

Comment:
Why we can't recreate parents instead of retriggering the event? Processor should not trigger events, it will add the event to the end of queue which might lead to huge issues as the task might change meanwhile.